src/button-group/styles.scss

```diff
@@ -0,0 +1,26 @@
+/*
+ Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ SPDX-License-Identifier: Apache-2.0
+*/
+@use '../internal/styles/tokens' as awsui;
+@use '../internal/styles' as styles;
+
+.root {
+  @include styles.styles-reset;
+}
+
+.item {
+  /* used in test-utils */
+}
+
+.group {
+  display: flex;
```

Comment:
This one does not maintain the consistent spacing between the items. We can add the matching gap and no-wrap but can try use `display: contents` instead. The group div is needed to assign the ARIA role and label, but we can exclude it from the layout effectively.

Reply:
Alternatively, we can keep it and use to display dividers which can be defined as inline-end border on the group. In that case we won't need a separate divider element.